### PR TITLE
Fix compatibility issue for dynamic feature

### PR DESCRIPTION
**Pull request by IgorGanapolsky**
Update versions in build.gradle files.

### CODE REVIEW

**Comment by keyboardsurfer** [comment]
Hi Igor and thanks for this PR. It introduces a lot of files that are not necessary to compile the project and contains a lot of changes. It's hard to find the spot where this actually fixes something.
Before I can evaluate a fix and potentially approve it this needs to be cleaned up.

**Comment by IgorGanapolsky** [comment]
@keyboardsurfer Do you advise adding **.gitignore** to this project?  That is the reason for so many files...

**Comment by keyboardsurfer** [comment]
I've got a global .gitignore file which I keep for all projects.
It contains things such as .idea folder and .iml files.

File: install-api/features/base/build.gradle [hunk 1]
```diff
@@ -18,7 +18,7 @@ apply plugin: 'com.android.feature'
 
 android {
     compileSdkVersion rootProject.compileSdk
-    buildToolsVersion rootProject.buildTools
+//    buildToolsVersion rootProject.buildTools
```
Comment (keyboardsurfer) [inline install-api/features/base/build.gradle:21]: Don't add scar tissue. Remove the commented out lines completely.

File: multi-feature-module/.idea/libraries/Gradle__android_arch_core_common_1_1_0_jar.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<component name="libraryTable">
```
Comment (keyboardsurfer) [inline multi-feature-module/.idea/libraries/Gradle__android_arch_core_common_1_1_0_jar.xml:1]: Remove .idea folder completely.

File: multi-feature-module/detail/detail.iml [hunk 3]
```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (keyboardsurfer) [inline multi-feature-module/detail/detail.iml:1]: Remove .iml files.

File: urlless/.gitignore [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# built application files
```
Comment (keyboardsurfer) [inline urlless/.gitignore:1]: One single .gitignore for the whole project should be enough. Keep it in the project root.

File: urlless/README.md [hunk 5]
```diff
@@ -10,7 +10,7 @@ ia run ./instantapp/build/outputs/apk/debug/instantapp-debug.zip
 ```
 
 ## How do I get 'ia' CLI?
-Install Instant Apps Development SDK (revision: 1.3.0 or higher) from Android SDK Manager.
+Install Instant Apps Development SDK (revision: 1.2.0) from Android SDK Manager.
```
Comment (keyboardsurfer) [inline urlless/README.md:13]: Revert

File: urlless/feature/src/main/java/com/google/android/instantapps/samples/urlless/feature/MainActivity.java [hunk 6]
```diff
@@ -18,8 +18,8 @@
```
Comment (keyboardsurfer) [inline urlless/feature/src/main/java/com/google/android/instantapps/samples/urlless/feature/MainActivity.java:18]: This is not necessary within this change.

File: urlless/local.properties [hunk 7]
```diff
@@ -0,0 +1,9 @@
+## This file must *NOT* be checked into Version Control Systems,
```
Comment (keyboardsurfer) [inline urlless/local.properties:1]: Remove file

**Review by keyboardsurfer — CHANGES_REQUESTED** [review]

**Comment by IgorGanapolsky** [comment]
I think the main issue is that git forces parent directories to be committed when using **.gitignore**.  For example, my changes to **urlless** directory inherently force changes upon **multi-feature-module**,  **install-api**, and other projects in the parent folder.  That is the way git is designed to work with .gitignore files (AFAIK).  So we have two options:
1) scrap this entire PR and start over
2) figure out a way to break these projects into truly separate repos, so that they don't step on each others' feet.

Please let me know if one of these solutions makes sense to you....

**Comment by keyboardsurfer** [comment]
I'm okay with closing this PR and starting over.
AFAIK .gitignore files don't traverse the tree upwards, so I suggest editing the file in the project root.

**Pull request closed without merging**